src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -20,10 +20,17 @@
            [org.apache.commons.fileupload.util LimitedInputStream]))
 
 (def min-supported-commands
```

Comment:
That was intentional from what I recall, I had the same comment when the code originally went in